contracts/root/Vesting.sol

```diff
@@ -139,7 +114,7 @@ contract Vesting is Ownable {
 
     function claimableAmount(address user) public view returns (uint256) {
```

Comment:
We better have another claimableAmount that doesn't count vtSQT balance. So user knows how many vtSQT they will need to claim all unlocked token.

Reply:
Do you mean `unlockedAmount - vtSQTAmount`, I suppose we will maintain all these values in subquery project, or the app can calculate with the above values, just avoid another RPC call